package.json

```diff
@@ -14,6 +14,7 @@
     "react-dom": "^17.0.1",
     "uuid": "^8.3.2"
   },
+  "sideEffects": false,
```

Comment:
Curious what this is for...

Reply:
@bobular  I was curious about that so I googled it to learn about the option :) Although you may find it easily, here is a summary. It is related to Webpack's optimization, so-called tree shaking. Simply put, Webpack tries to exclude unused external library/package to minimize its compiled file size (optimization), but certainly it often makes codes not work correctly, so-called side effects. The optimization is performed when the sideEffects is set to false (meaning side effects are free), so any side effects by Webpack's optimization become developers' responsibility. Actually, the option can be set at a library's package.json too.

Certainly it is more complex than what I said earlier: please see https://webpack.js.org/guides/tree-shaking/ if you are interested